test/support/Try.js

```diff
@@ -0,0 +1,40 @@
+
+import bluebird from 'bluebird';
+
+import {
+  defaultTryTimeout
+} from './index';
```

Comment:
I would like to move this file into the utils directory, but this dependency upon index needs to be removed first.